### ISSUE TITLE
Exception during "Packaging required Python packages..."

### ISSUE THREAD

**Issue by whitebarry** [logandk/serverless-wsgi#106]
Hello, I'm trying to follow the flask example but I'm getting the following output:

```
build	27-Aug-2019 14:53:06	Serverless: Successfully installed "serverless-wsgi@latest"
build	27-Aug-2019 14:53:07	Serverless: Using Python specified in "runtime": python3.6
build	27-Aug-2019 14:53:07	Serverless: Packaging Python WSGI handler...
build	27-Aug-2019 14:53:07	Serverless: Packaging required Python packages...
build	27-Aug-2019 14:53:07	 
build	27-Aug-2019 14:53:07	 Exception -----------------------------------------------
build	27-Aug-2019 14:53:07	 
build	27-Aug-2019 14:53:07	  <Buffer 55 6e 61 62 6c 65 20 74 6f 20 6c 6f 61 64 20 76 69 72 74 75 61 6c 65 6e 76 2c 20 70 6c 65 61 73 65 20 69 6e 73 74 61 6c 6c 0a>
build	27-Aug-2019 14:53:07	 
build	27-Aug-2019 14:53:07	     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.
build	27-Aug-2019 14:53:07	 
build	27-Aug-2019 14:53:07	  Get Support --------------------------------------------
build	27-Aug-2019 14:53:07	     Docs:          docs.serverless.com
build	27-Aug-2019 14:53:07	     Bugs:          github.com/serverless/serverless/issues
build	27-Aug-2019 14:53:07	     Issues:        forum.serverless.com
build	27-Aug-2019 14:53:07	 
build	27-Aug-2019 14:53:07	  Your Environment Information ---------------------------
build	27-Aug-2019 14:53:07	     Operating System:          linux
build	27-Aug-2019 14:53:07	     Node Version:              10.15.1
build	27-Aug-2019 14:53:07	     Framework Version:         1.50.1
build	27-Aug-2019 14:53:07	     Plugin Version:            1.3.9
build	27-Aug-2019 14:53:07	     SDK Version:               2.1.0
```

Enabling the debug logs just seems to create some unrelated clutter with this same message repeating many times:
```
build	27-Aug-2019 14:46:30	Serverless: 
build	27-Aug-2019 14:46:30	User stats error: Request network error: { FetchError: request to https://serverless.com/api/framework/track failed, reason: read ECONNRESET
build	27-Aug-2019 14:46:30	    at ClientRequest.<anonymous> (/home/bamboo/.nvm/versions/node/v10.15.1/lib/node_modules/serverless/node_modules/node-fetch/index.js:133:11)
build	27-Aug-2019 14:46:30	    at ClientRequest.emit (events.js:189:13)
build	27-Aug-2019 14:46:30	    at ClientRequest.EventEmitter.emit (domain.js:441:20)
build	27-Aug-2019 14:46:30	    at TLSSocket.socketErrorListener (_http_client.js:392:9)
build	27-Aug-2019 14:46:30	    at TLSSocket.emit (events.js:189:13)
build	27-Aug-2019 14:46:30	    at TLSSocket.EventEmitter.emit (domain.js:441:20)
build	27-Aug-2019 14:46:30	    at emitErrorNT (internal/streams/destroy.js:82:8)
build	27-Aug-2019 14:46:30	    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
build	27-Aug-2019 14:46:30	    at process._tickCallback (internal/process/next_tick.js:63:19)
build	27-Aug-2019 14:46:30	  name: 'FetchError',
build	27-Aug-2019 14:46:30	  message:
build	27-Aug-2019 14:46:30	   'request to https://serverless.com/api/framework/track failed, reason: read ECONNRESET',
build	27-Aug-2019 14:46:30	  type: 'system',
build	27-Aug-2019 14:46:30	  errno: 'ECONNRESET',
build	27-Aug-2019 14:46:30	  code: 'ECONNRESET' }
```

My serverless.yml:
```yml
service: flaskLambdaExample

provider:
  name: aws
  runtime: python3.6

plugins:
  - serverless-wsgi

functions:
  api:
    handler: wsgi_handler.handler
    events:
      - http: ANY /
      - http: ANY {proxy+}

custom:
  wsgi:
    app: src.main.app
```

requirements.txt:
```
flask==1.1.1
```

Any help would be greatly appreciated!

**Comment by mikenotfound** [comment]
I had the same thing. Running `strace -s 128 -f  sls package -v > strace.txt` I spotted it was complaining about being unable to import virtualenv.

`pip install virtualenv` got me going again.

**Comment by whitebarry** [comment]
@mikenotfound Thanks for the suggestion. Before I saw it, I ended up switching my project up and using the `serverless-python-requirements` plugin instead and just used the `serverless-wsgi` python package to have access to the handler. That ended up working out perfectly after a little tweaking to `serverless.yml`. I'll look into my build environment anyway to ensure virtualenv is available. 

I'm going to close this on the assumption that this is just an environmental issue for me.
